store/src/java-test/com/zimbra/cs/filter/RuleManagerAdminFilterTest.java

```diff
@@ -186,22 +185,17 @@ public void invalidRequireComand() throws Exception {
     String[] variableScripts = {
             // admin-before
               "require [\"tag\", \"log\", \"variables\"];"
-            + "set \"beforevar\" \"foo\";"
-            + "tag \"admin-before1-${beforevar}\";",
+            + "set \"var\" \"foo\";"
```

Comment:
You try to back-out the changes done by zcs-1378!   This RuleManagerAdminFilterTest test works fine now.